Runtime/Tracking/Velocity/VelocityEmitter.cs

```diff
@@ -20,9 +20,16 @@ public class VelocityEmitter : MonoBehaviour
         public class UnityEvent : UnityEvent<Vector3>

```

Comment:
I wonder if this should be renamed to Vector3UnityEvent as there is now a FloatUnityEvent as well

Reply:
Would that be a breaking change?

Reply:
And i feel like that rename shall be a separate chore PR

Reply:
No i don't think it would be a breaking change as it's still just internal to the class itself, changing the name of the internal unity class _shouldn't_ break anything elsewhere.

I don't think it needs to be a separate PR either. I dunno if @bddckr has time to have a view on this.

Reply:
I can access the computer to check on this after few hours

Reply:
I'm not sure how Unity handles nested types in their serialization engine, so better check to see if it's a breaking change or not. :+1: